flask-server/main.py

```diff
@@ -263,6 +231,37 @@ def check_denylist(token):
     else:
         return False
 
+# add clink api
+@app.route('/apis/add-clink', methods=['POST'])
+def add_clink():
```

Comment:
Could be in another PR, but in this method we should also validate user's JWT token